hera_cal/abscal.py

```diff
@@ -1544,21 +1769,37 @@ def _put_transformed_array_on_integer_grid(transformed_antpos, tol=1e-8, max_num
 
         # If any position not on integer grid
         if np.max(np.abs(np.rint(this_dim_positions) - this_dim_positions)) > tol:
-
             # Look for multipliers that put every antenna on an integer grid
             for numerator in range(2, max_numerator + 1):
-                if np.max(np.abs(np.rint(this_dim_positions * numerator) - this_dim_positions * numerator)) < tol:
+                if (
+                    np.max(
+                        np.abs(
+                            np.rint(this_dim_positions * numerator) - this_dim_positions * numerator
+                        )
+                    )
+                    < tol
+                ):
```

Comment:
this is another spacious re-lining. probably nicer if we save the interior thing as a variable on its own line.